vhd/packer/install-dependencies.sh

```diff
@@ -497,6 +497,15 @@ for CSI_SECRETS_STORE_DRIVER_VERSION in ${CSI_SECRETS_STORE_DRIVER_VERSIONS}; do
   echo "  - ${CONTAINER_IMAGE}" >> ${VHD_LOGS_FILEPATH}
 done
 
+CLUSTER_PROPORTIONAL_AUTOSCALER_VERSIONS="
+1.7.1
```

Comment:
So we weren't adding this to the VHD at all? I'll try to reconcile calico's version with this one or add it here after this merges.

Reply:
Good call